oauth2client/contrib/sql_alchemy.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""OAuth 2.0 utilities for SQLAlchemy.
+Utilities for using OAuth 2.0 in conjunction with SQLAlchemy.
+Heavily inspired by equivalent Django module.
+"""
+
+from oauth2client.client import Storage as BaseStorage
+from sqlalchemy.types import PickleType
+
+
+__author__ = 'dominik@mdzn.pl (Dominik Miedzinski)'
+
+
+class CredentialsType(PickleType):
+    pass
+
+
+class FlowType(PickleType):
+    pass
+
+
+class Storage(BaseStorage):
+    """Store and retrieve a single credential to and from SQLAlchemy.
+    This Storage helper presumes the Credentials
+    have been stored as a Credentials column
```

Comment:
This reads strangely to me. I think this is better documented in the constructor - see my comment below.